shared/lib/ortcustomops.cc

```diff
@@ -6,6 +6,7 @@
 
 #include "onnxruntime_extensions.h"
 #include "ocos.h"
+#include "vision/vision.hpp"
```

Comment:
Did you check whether if the entire source code related to PPP_vision will be excluded if PPP_VISION was disabled by CMAKE argument?

Reply:
If it's disabled the factory will have no classes and the custom ops will have no usage. When that happens the linker should be able to throw them away. At least that's how it works in ORT where we just remove the registration of kernels when doing a reduced ops build (we don't need to remove the code for the kernels).

If you think something more is required I can add it.

Reply:
The problem is the 3rd party library will not be fetched when some operators are disabled. So, the file including statement will cause some compiling issue. This is different from ORT since it uses submodule.

BTW, I am not sure whether there will be some binary increase if we just include a 3rd party library and do not reference any functions from it. Is C++ compiler smart enough to handle the global variable issue in that case?

Reply:
I can wrap the include of the vision.hpp header in an ifdef if you want. Shouldn't matter though. There are no external includes in the vision custom op headers. I assumed there wouldn't be any compile issues from including ocos.h or string_utils.h. Is that the case? 

As for the binary size it might depend on the 3rd party library implementation as to whether the linker can throw it all away. Easily tested once we create a package with the desired operators. Ideally the cmake setup isn't adding unused 3rd party libraries to the link list. If it is it would be better to fix that.